samples/JsonPatchSample.Web/Models/Category.cs

```diff
@@ -3,15 +3,8 @@
 
 using Newtonsoft.Json;
 
-namespace JsonPatchWebSite.Models
+namespace JsonPatchSample.Web.Models
 {
-    public class Product
-    {
-        public string ProductName { get; set; }
-
-        public Category ProductCategory { get; set; }
-    }
-
```

Comment:
FYI I split this out into its own file. Did that with one other class somewhere.